src/pages/settings/Payments/TransferBalancePage.js

```diff
@@ -30,7 +30,7 @@ const propTypes = {
     userWallet: userWalletPropTypes.userWallet,
 
     /** Array of bank account objects */
```

Comment:
NAB: Ah I think we should update that this is no longer an array but an object, no?